Controllers/BaseController.php

```diff
@@ -1,20 +1,39 @@
 <?php
 
+declare(strict_types=1);
+
 namespace Controllers;
 
+/**
+ * @package Controllers
+ * Base controller for extends with base methods
+ */
 class BaseController
 {
-    protected function checkPost()
+    /**
+     * Check for valid array POST
+     * @return bool
+     */
+    protected function checkPost(): bool
     {
         return $_SERVER['REQUEST_METHOD'] == 'POST' && !empty($_POST);
```

Comment:
use server/session and request models instead of working with core php variables